Documentation/MigrateToComposer/MigrationSteps.rst

```diff
@@ -376,43 +436,36 @@ Move files:
    :file:`public/typo3temp/var`, unless you need to keep the log files
    or anything else that may still be relevant.
 
-These locations have changed:
-
-+--------------------------------+-----------------------+
-| Before                         | After                 |
-+================================+=======================+
-| :file:`public/typo3conf/sites` | :file:`config/sites`  |
-+--------------------------------+-----------------------+
-| :file:`public/typo3temp/var`   | :file:`var`           |
-+--------------------------------+-----------------------+
-| :file:`public/typo3conf/l10n`  | :file:`var/labels`    |
-+--------------------------------+-----------------------+
-
+These locations have changed, note that TYPO3 v12+ moved some more configuration
+files to a new directory than TYPO3 v11:
+
++------------------------------------------------------+-----------------------------------------------------------------+
+| Before                                               | After                                                           |
++======================================================+=================================================================+
+| :file:`public/typo3conf/sites`                       | :file:`config/sites`                                            |
++------------------------------------------------------+-----------------------------------------------------------------+
+| :file:`public/typo3temp/var`                         | :file:`var`                                                     |
```

Comment:
Personally, I would remove the var and l10n folders completely, they are created again (also after fetching the translations).